javascript/src/main/components/Statistics/FullCourseTable.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import Badge from 'react-bootstrap/Badge'
+import Container from 'react-bootstrap/Container'
+
+const white = {
+  backgroundColor: '#ffffff'
+};
+
+const d2d4d7 = {
+  backgroundColor: '#d2d4d7'
+}
```

Comment:
Instead of
```
const d2d4d7 = {
  backgroundColor: '#d2d4d7'
}
``` 

I wonder if it would be better to give this color a semantically meaningful name?  How is it used in the code?  

e.g. 
```
const sectionColor = ...
const lectureColor = ...
const fullSectionColor = ...
```

Reply:
Updated!